Dockerfile

```
@@ -0,0 +1,5 @@
+FROM ruby:slim-buster
+
+WORKDIR /app
+COPY . .
+CMD ["./server"]
```

This will make it bind to 0.0.0.0 only in the docker container and we can then remove the "bind" line from the "server" file
```suggestion
CMD ["./server", "-o", "0.0.0.0"]
```